src/scenes/Home/ParticipantHomeView/Header.tsx

```diff
@@ -43,12 +43,16 @@ const Header = () => {
       <Grid item marginTop={theme.spacing(1)} marginLeft={theme.spacing(1)}>
         <Grid container alignItems={'center'} columnSpacing={theme.spacing(2)}>
           <Grid item>
-            <Dropdown
-              onChange={setCurrentParticipantProject}
-              options={options}
-              selectStyles={selectStyles}
-              selectedValue={currentParticipantProject?.id}
-            />
+            {options?.length > 1 ? (
+              <Dropdown
+                onChange={setCurrentParticipantProject}
+                options={options}
+                selectStyles={selectStyles}
+                selectedValue={currentParticipantProject?.id}
+              />
+            ) : (
+              <Typography sx={selectStyles.input}>{options[0].label}</Typography>
```

Comment:
If there is only 1 option, should it also be auto selected?

Reply:
Yes, but it's already selected

Reply:
Oh I see, thanks